# HG changeset patch
# User Rik <rik@octave.org>
# Date 1465604607 25200
#      Fri Jun 10 17:23:27 2016 -0700
# Node ID bfc662ee0340d67b5cca5615a69c862afd6ddd61
# Parent  ce0f87bd05095f28aae84991cb5c4f0da0f5941f
Improve docstring for system().

* toplev.cc (Fsystem): Change example to discard unused output of system call
when running the output text through the pager.

diff --git a/libinterp/corefcn/toplev.cc b/libinterp/corefcn/toplev.cc
--- a/libinterp/corefcn/toplev.cc
+++ b/libinterp/corefcn/toplev.cc
@@ -1084,17 +1084,17 @@ If @var{system} is called with one or mo
 optional argument @var{return_output} is true and the subprocess is started\n\
 synchronously, then the output from the command is returned as a variable.\n\
 Otherwise, if the subprocess is executed synchronously, its output is sent\n\
 to the standard output.  To send the output of a command executed with\n\
 @code{system} through the pager, use a command like\n\
 \n\
 @example\n\
 @group\n\
-[output, text] = system (\"cmd\");\n\
+[~, text] = system (\"cmd\");\n\
 disp (text);\n\
 @end group\n\
 @end example\n\
 \n\
 @noindent\n\
 or\n\
 \n\
 @example\n\
